app.config.ts

```diff
@@ -0,0 +1,14 @@
+import { ExpoConfig } from 'expo/config';
+import { withSentry } from '@sentry/react-native/expo';
+
+const config: ExpoConfig = {
+  name: 'Rainbow',
+  slug: 'rainbowdotme',
+};
+
+export default withSentry(config, {
```

Comment:
as per: https://docs.sentry.io/platforms/react-native/sourcemaps/uploading/expo/#add-the-sentry-expo-plugin